bitnami/thanos/README.md

```diff
@@ -1286,6 +1286,8 @@ Thanos store supports partion based on time.
 
 Setting time partitions will create N number of store statefulsets based on the number of items in the `timePartitioning` list. Each item must contain the min and max time for querying in the supported format (find more details at [Thanos documentation](https://thanos.io/tip/components/store.md/#time-based-partitioning)).
 
+You can speficiy different resources and limits configurations to each storegateway statefulset. This is done by adding a `resources.requests` and `resources.limits` to each item. 
+
```

Comment:
I think adding an example of this in the example shown below would be nice.
(Like the example you posted in the PR description).

Reply:
Hi @rafariossaa thanks for you comment. I agree. Added the example